sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsExtensionConfigProvider.cs

```diff
@@ -205,6 +216,51 @@ public async Task<HttpResponseMessage> ConvertAsync(HttpRequestMessage input, Ca
             return (T)blob.BlobClient;
         }
 
+        private ParameterBindingData ConvertToParameterBindingData(BlobAttribute blobAttribute)
+        {
+            var blobPath = BlobPath.ParseAndValidate(blobAttribute.BlobPath);
+            return CreateParameterBindingData(blobAttribute.Connection, blobPath.BlobName, blobPath.ContainerName);
+        }
+
+        private ParameterBindingData ConvertToParameterBindingData(BlobBaseClient input, BlobTriggerAttribute blobTriggerAttribute)
+        {
+            return CreateParameterBindingData(blobTriggerAttribute.Connection, input.Name, input.BlobContainerName);
+        }
+
+        private ParameterBindingData ConvertBlobInputToParameterBindingData(BlobBaseClient input)
+        {
+            return CreateParameterBindingData(null, input.Name, input.BlobContainerName);
+        }
+
+        private ParameterBindingData CreateParameterBindingData(string connection, string blobName, string containerName)
+        {
+            var connectionString = ResolveConnection(connection);
+
+            var blobDetails = new BlobParameterBindingDataContent()
+            {
+                Connection = connectionString,
+                BlobName = blobName,
+                ContainerName = containerName
+            };
+
+            var blobDetailsBinaryData = new BinaryData(blobDetails);
+            var bindingData = new ParameterBindingData("1.0", Constants.WebJobsBlobExtensionName, blobDetailsBinaryData, "application/json");
+            return bindingData;
+        }
+
+        private string ResolveConnection(string connection)
+        {
+            string connectionName = !string.IsNullOrEmpty(connection) ? _nameResolver.ResolveWholeString(connection) : Constants.DefaultAzureStorageConnectionName;
```

Comment:
We don't need to define a new constant for DefaultAzureStorageConnectionName - you can just use the public ConnectionStringNames value, as is done here: https://github.com/Azure/azure-sdk-for-net/blob/4162f6fa2445b2127468b9cfd080f01c9da88eba/sdk/extensions/Microsoft.Azure.WebJobs.Extensions.Clients/src/Shared/StorageClientProvider.cs#L72.

Actually at this point, you've created a duplicate of the code I linked to. We should define a new GetWebJobsStorageConnectionSection method that both places call, so we're centralizing this ConnectionStringNames.Storage defaulting and error handling.

Reply:
Done